test_utils/utils/setup_fixtures.py

```diff
@@ -300,3 +300,19 @@ def create_employees(settings, only_create=None):
 					empl.employee_primary_address = addr.name
 					empl.save()
 			break
+
+
+def create_holiday_lists(settings):
+	holiday_lists = get_fixtures_data_from_file(filename="holiday_lists.json")
```

Comment:
I'd like to detect the countries of the companies installed to check against the data.